docs/developer-guide.md

```diff
@@ -0,0 +1,42 @@
+# Developer guide
+
+Have a look at the [user guide](user-guide.md) to familiarize yourself with things like rule naming conventions.
+
+## Running tests
+
+You can run the tests via:
+
+```console
+npm test
+```
```

Comment:
You can even run `babel-tape-runner src/rules/media-feature-name-no-vendor-prefix/__tests__` that should work